src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,136 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Scanner;
+
+/**
+ * A class that creates the Storage object.
+ */
+public class Storage {
```

Comment:
Could help if there are separate TaskList encoder and decoder classes to handle conversion of tasks to text and back.